amgut/db/patches/0009.sql

```diff
@@ -0,0 +1,229 @@
+--Create pet survey in database
+----------------------------------------------------------
+-- survey_group
+----------------------------------------------------------
+INSERT INTO survey_group (group_order, american, british) VALUES (-2, 'Pet Information', 'Pet Information');
+
+----------------------------------------------------------
+-- surveys
+----------------------------------------------------------
+INSERT INTO surveys (survey_id, survey_group) VALUES (2, -2);
+
+----------------------------------------------------------
+-- survey_question
+----------------------------------------------------------
+INSERT INTO survey_question (survey_question_id, question_shortname, american, british) VALUES (127, 'NAME', 'Name', 'Name');
```

Comment:
...actually, if there isn't a `UNIQUE` constraint already then we may want to add one for shortname